README.md

```diff
@@ -12,11 +12,19 @@ The Guides' content is in the `source` folder. The left nav bar is produced from
 `data/pages.yml`. `lib` contains Middleman plugins, and `spec` contains tests
 for those plugins.
 
-## Developing with the Guides
+## Dependencies
 
 The Guides are built with Middleman, which runs on Ruby 1.9.3 or newer
 (2.0.0 recommended).
 
+During build, Middleman will require Aspell to look for misspellings. It can be installed via Homebrew:
+
+``` sh
+brew install aspell --with-lang-en
+```
+
```

Comment:
During build, Middleman will require Aspell to look for misspellings. On Macs, it can be installed via [Homebrew](http://brew.sh/):

``` sh
brew install aspell --with-lang-en
```

On Windows, you can [download an installer](http://aspell.net/win32/), but unfortunately it is unmaintained. On Linux, you can install with your distribution's package manager. On all platforms, you can also [build the most recent version from source](http://aspell.net/man-html/Installing.html).